robottelo/constants/__init__.py

```diff
@@ -1656,6 +1656,31 @@ class Colored(Box):
     'Viewer',
 ]
 
+BOOKMARK_ENTITIES_SELECTION = [
```

Comment:
Most components you have selected are from content-management pillar of satellite. 

Would you like to change the horizon to cover provisioning, subscription, compliance etc ?

Reply:
I added a few from provisioning area, we didn't have much in compliance and subscription sections